modules/languages/php.nix

```diff
@@ -0,0 +1,100 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib;
+with builtins; let
+  cfg = config.vim.languages.php;
+
+  defaultServer = "phpactor";
+  servers = {
+    phpactor = {
+      package = pkgs.phpactor;
+      lspConfig = ''
+        lspConfig.phpactor.setup{
```

Comment:
typo, `lspconfig` - c should not be capitalized

Reply:
https://github.com/NotAShelf/neovim-flake/blob/fc651e654f736b2b89f268347726273211c6fc25/modules/languages/go.nix#L122C6-L122C6

I took example there though, it is capitalized here

Reply:
I meant the lua code, `lspconfig.phpactor.setup {`

Reply:
Sorry didnt see it